test/chunking-form/samples/dynamic-import-chained/_expected/amd/generated-dep1.js

```diff
@@ -1,6 +1,12 @@
-define(['require'], function (require) { 'use strict';
+define(function () { 'use strict';
+
+	console.log('dep2');
```

Comment:
this is a case where the execution order has changed. Not sure if this is something that should be expected or if this would be breaking